packages/sdk/src/testing.rs

```diff
@@ -32,14 +30,15 @@ pub type AppBuilder<Exec = CustomMsg, Query = Empty> = cw_multi_test::AppBuilder
     FailingModule<GovMsg, Empty, Empty>,
 >;
 
-pub type Contract = dyn cw_multi_test::Contract<CustomMsg>;
+pub type CwContract = dyn cw_multi_test::Contract<CustomMsg>;
 
 pub type CustomMessageSender = std::sync::mpsc::Sender<CustomMsg>;
 type CustomMessageReceiver = std::sync::mpsc::Receiver<CustomMsg>;
 
-pub struct WrappedCustomMessageReceiver(CustomMessageReceiver);
+#[derive(Debug)]
+pub struct CustomMessageReceiverExt(CustomMessageReceiver);
```

Comment:
I respectfully disagree because this is both outside our business logic and not necessarily a DEX message. I am willing to move the methods within a trait that extends the receiver, but not calling it "Dex" related.